lib/components/embed.js

```diff
@@ -1,20 +1,26 @@
 import element from 'magic-virtual-element';
 import renderText from '../text';
 
-const defaultRenderCaption = text => <figcaption>{text}</figcaption>;
+const defaultRenderCaption = (text, attribution) => {
+  const attributionEl = attribution.length > 0 ? <cite>{attribution}</cite> : '';
+  return (<figcaption>
+    {text.concat([attributionEl])}
```

Comment:
Ok, so weird. Let's go with this now then and then see if we can change it somehow later on :)